packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -18,7 +18,10 @@ export function ensureInitialized(...expectedFrameContexts: string[]): void {
     }
 
     if (!found) {
-      throw new Error("This call is not allowed in the '" + GlobalVars.frameContext + "' context");
+      throw new Error(
+        `This call is only allowed in following contexts: ${JSON.stringify(expectedFrameContexts)}. ` +
+          `Current context: "${GlobalVars.frameContext}".`,
```

Comment:
If portions of this string could be made into a constant and shared between this code and the tests, it would likely aid future maintainability. Possibly even sharing the function that generates the string? Something to consider.

Reply:
I'll leave it as is for now, if you don't mind.